src/language-server/project/rover/project.ts

```diff
@@ -150,7 +150,18 @@ export class RoverProject extends GraphQLProject {
         "Connection is closed.",
       );
     }
-    const child = cp.spawn(this.config.rover.bin, ["lsp"], {
+    const args = ["lsp", "--elv2-license", "accept"];
```

Comment:
@dylan-apollo could you please verify that I understood you correctly and this should always be added? :)

Reply:
Yep!